clusters/app.ci/prow/03_deployment/admin_hook_rbac.yaml

```diff
@@ -1,73 +1,6 @@
 kind: List
 apiVersion: v1
 items:
-- apiVersion: v1
-  kind: ServiceAccount
-  metadata:
-    name: hook
-    namespace: ci
-- kind: Role
-  apiVersion: rbac.authorization.k8s.io/v1
-  metadata:
-    name: hook
-    namespace: ci
-  rules:
-    - apiGroups:
-        - prow.k8s.io
```

Comment:
Same here, access to prowjobs should only be granted on app.ci so we should split this role up in one for app.ci and one for all build clusters